roles/agnosticv/tasks/generate_catalog_item_tasks.yml

```diff
@@ -9,6 +9,25 @@
     template_namespace: >-
       {{ merged_vars.__meta__.catalog.namespace | default('openshift') }}
 
+- name: Get k8s fact for the catalog item
+  k8s_facts:
+    api_version: babylon.gpte.redhat.com/v1
+    kind: catalogitems
+    name: "{{ catalog_item_name }}"
+    namespace: "{{ template_namespace }}"
+  register: catalogitems_facts
+
+- name: Generate Catalog item (CR)
```

Comment:
What is `CR` referencing?

Reply:
short for custom resource
i'll make it explicit